operations/kubernetes/Repopulate_Data_in_etcd_Clusters_When_Rebuilding_Them.md

```diff
@@ -39,7 +39,7 @@ Repopulate clusters for CPS.
 
 ## CRUS
 
-**Note:** CRUS is deprecated in CSM 1.2.0. It will be removed in a future CSM release and replaced with BOS V2, which will provide similar functionality. See
+> **`NOTE`** CRUS was deprecated in CSM 1.2.0. It will be removed in a future CSM release and replaced with BOS V2, which will provide similar functionality. See
```

Comment:
```suggestion
**Note:** CRUS is deprecated in CSM 1.2.0. It will be removed in a future CSM release and replaced with BOS V2, which will provide similar functionality. See
```